deploy_windows_files.ps1

```diff
@@ -49,14 +49,20 @@ if ($clean_deployment_folder) {
     Invoke-RemoteCommand -Command $clean -Arguments $deployment_folder_path
 }
 
-[Byte[]]$zip = Get-Content -Path $source_zip_file_path -Encoding Byte
-$zip_size = (Get-Item -Path $source_zip_file_path).Length / 1KB
+Write-Output "Copy file: $source_zip_file_path"
+
+$copy_session = New-PSSession $server -SessionOption $so -UseSSL -Credential $credential
+Copy-Item -Path $source_zip_file_path -ToSession $copy_session -Destination $destination_zip_file_path
+
+# [Byte[]]$zip = Get-Content -Path $source_zip_file_path -Encoding Byte
+# $zip_size = (Get-Item -Path $source_zip_file_path).Length / 1KB
+# Write-Output "Zip Size: $zip_size"
 
 $copy = {
-    param([string]$path, [string]$file, [Byte[]]$zip_data, [int]$file_size)
-    Write-Host "Writing Package Archive: $file"
-    Write-Host "File Size: $file_size KB"
-    Set-Content -Path $file -Value $zip_data -Encoding Byte
+    param([string]$path, [string]$file) #, [Byte[]]$zip_data, [int]$file_size)
+    # Write-Host "Writing Package Archive: $file"
+    # Write-Host "File Size: $file_size KB"
+    # Set-Content -Path $file -Value $zip_data -Encoding Byte
```

Comment:
Did you want to keep all of these comments? If so, do you think a comment would be good to explain why (so others don't have my same question)?

Reply:
I forgot to check-in my comments clean up...should be better now.